lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -266,8 +270,23 @@ describe('AwsInvokeLocal', () => {
       })
     );
 
-    it('throw error when using runtime other than Node.js', () => {
-      awsInvokeLocal.options.functionObj.runtime = 'python';
+    it('should call invokeLocalPython when python2.7 runtime is set', () => {
```

Comment:
Yes, there's a check for the runtime which matches the AWS runtime definition (therefore python 2.7).